service/controller/legacy/v22/resource/ipam/create.go

```diff
@@ -10,13 +11,17 @@ import (
 	"sync"
 	"time"
 
+	clusterv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/cluster/v1alpha1"
```

Comment:
For naming aliases I found https://github.com/giantswarm/aws-operator/blob/master/service/controller/clusterapi/v27/key/cluster_provider.go#L3-L9 pretty good.